packages/grid/x-data-grid/src/colDef/gridSingleSelectColDef.tsx

```diff
@@ -54,4 +54,25 @@ export const GRID_SINGLE_SELECT_COL_DEF: Omit<GridSingleSelectColDef, 'field'> =
   },
   renderEditCell: renderEditSingleSelectCell,
   filterOperators: getGridSingleSelectOperators(),
+  // @ts-ignore
```

Comment:
The `release:build` would fail without it, because `pastedValueParser` is defined in the Premium package.
I believe `yarn typescript` doesn't fail without it, because it applies type augmentations globally for the whole codebase.

Reply:
Ah ok 👍